lib/sync/mongodbQueue.js

```diff
@@ -0,0 +1,68 @@
+var mongodbQ = require('mongodb-queue');
```

Comment:
@wei-lee you can probably use this in sync/index.js now. There's placeholders for where queues are created directly with mongodb-queue already https://github.com/feedhenry/fh-mbaas-api/blob/FH-3123-sync-horizontal-scaling/lib/sync/index.js#L121-L125